### PR TITLE
Update coarse-parallel-processing-work-queue.md

### PR DESCRIPTION
**Pull request by VineethReddy02**
Considering the consistency among shell scripts. Differentiated inputs and outputs for better readability and understanding.

### CODE REVIEW

**Comment by VineethReddy02** [comment]
after we have clear guidelines about code formatting based on https://github.com/kubernetes/website/pull/12783 i will reopen the PR

**Comment by VineethReddy02** [comment]
I think this PR is for better visibility so considering user readability would like to reopen the PR.
Thanks!

File: content/en/docs/tasks/job/coarse-parallel-processing-work-queue.md [hunk 1]
```diff
@@ -45,10 +45,20 @@ cluster and reuse it for many jobs, as well as for long-running services.
 
 Start RabbitMQ as follows:
 
+Step: 1
 ```shell
 $ kubectl create -f examples/celery-rabbitmq/rabbitmq-service.yaml
```
Comment (DanyC97) [inline content/en/docs/tasks/job/coarse-parallel-processing-work-queue.md]: please remove `$` as per recent style guideline changes

Comment (VineethReddy02) [reply content/en/docs/tasks/job/coarse-parallel-processing-work-queue.md]: @DanyC97  I made removed $ can u review the changes.

**Comment by DanyC97** [comment]
@VineethReddy02 you mind fixing the conflicting file ?

**Comment by mistyhacks** [comment]
Please reopen and resolve the conflict if you are still interested in getting this into the documentation. Thanks!

**Pull request closed without merging**